Workbooks/Workloads/MySQL Insights/MySQL Insights.workbook

```diff
@@ -440,7 +440,7 @@
                   "name": "SuggestedAlerts",
                   "type": 1,
                   "isRequired": true,
-                  "value": "[     {         \"AlertName\": \"host-high-cpu\",         \"AlertDescription\": \"Fires when high CPU usage is detected on the host\",         \"AlertThreshold\": 70,         \"AlertTemplate\": \"https://raw.githubusercontent.com/acearun/managedsolutions/master/Templates/Alerts/Host/Cpu/azuredeploy.json\"     },     {         \"AlertName\": \"host-used-memory-percentage\",         \"AlertDescription\": \"Fires when Used Memory percentage on the host exceeds a threshold\",         \"AlertThreshold\": 85,         \"AlertTemplate\": \"https://raw.githubusercontent.com/acearun/managedsolutions/master/Templates/Alerts/Host/Memory/azuredeploy.json\"     } ]",
+                  "value": "[     {         \"AlertName\": \"host-high-cpu\",         \"AlertDescription\": \"Fires when high CPU usage is detected on the host\",         \"AlertThreshold\": 70,         \"AlertTemplate\": \"https://raw.githubusercontent.com/acearun/managedsolutions/master/Templates/Alerts/Host/Cpu/azuredeploy.json\"     },     {         \"AlertName\": \"host-used-memory-percentage\",         \"AlertDescription\": \"Fires when Used Memory percentage on the host exceeds a threshold\",         \"AlertThreshold\": 85,         \"AlertTemplate\": \"https://raw.githubusercontent.com/acearun/managedsolutions/master/Templates/Alerts/Host/Memory/azuredeploy.json\"     },     {         \"AlertName\": \"MySQL slow queries\",         \"AlertDescription\": \"Fires when slow queries exceeds threshold\",         \"AlertThreshold\": 0,         \"AlertTemplate\": \"https://azmonsolutions.blob.core.windows.net/workloadinsights/Alerts/MySQL/mysql-slow-queries.json\"     } ]",

```

Comment:
Can we switch the memory alert to Available memory instead of Used memory?

Reply:
Zero threshold for slow queries seems hash. Is that the general recommendation?

Reply:
At some point, we need to find a better home for the alert ARM templates that the personal GitHub.

Reply:
Better to change the Host Alert as part of moving the alert to be based on the VM Insights Counter

Alerting on 1 slow query is what @austonli  wanted - we discussed this may not work on a production DB with lots of load, I think need additional feedback. It will only alert if in 3 5 minute intervals, so should not be noisy for something like quick maintenance.

I updated to point to the SA for the other Alerts.